tests/unit/given.test.scala

```diff
@@ -3,11 +3,11 @@
 
     given Foo = ???
 //  ^^^^^ keyword.declaration.scala
-//        ^^^ entity.name.type.declaration
+//        ^^^ - entity.name.type.declaration
   
     given foo = ???
 //  ^^^^^ keyword.declaration.scala
-//        ^^^ entity.name.declaration
+//        ^^^ - entity.name.declaration
```

Comment:
I think it would be better to test what it is, rather than what it isn't; this would allow us to better guard this against potential regressions. Right now we don't test what tokens `foo` or `Foo` are in these situations.

Reply:
This one is only `source.scala`